node/src/components/consensus/era_supervisor.rs

```diff
@@ -181,47 +180,32 @@ where
             era_where_we_joined: current_era,
         };
 
-        let bonded_eras = era_supervisor.bonded_eras();
-        let era_ids: Vec<EraId> = era_supervisor
-            .iter_past(current_era, bonded_eras.saturating_mul(3))
-            .collect();
-
         // Collect the information needed to initialize all recent eras.
-        // TODO: Merge these into one
-        let mut key_blocks = BTreeMap::new();
-        let mut booking_blocks = BTreeMap::new();
-        for era_id in era_ids {
-            let minimum_era = era_supervisor
-                .protocol_config
-                .last_emergency_restart
-                .unwrap_or_else(|| EraId::from(0u64));
-            let key_block_era_id = era_id.saturating_sub(1);
-            let block_header = storage
-                .read_switch_block_header_by_era_id(key_block_era_id)?
+        let auction_delay = era_supervisor.protocol_config.auction_delay;
+        let last_activation_point = era_supervisor.protocol_config.last_activation_point;
+        let mut switch_blocks = Vec::new();
+        // We need to initialize current_era, current_era - 1 and current_era - 2. To initialize an
```

Comment:
Consider changing to:

`// We need to initialize current_era, current_era - 1 and (frozen) current_era - 2. To initialize an`

Reply:
Not sure if usage of word "frozen" makes it more clear. What does it even mean?

Reply:
I'll add an explaining comment to the `active_eras` field:
So far I've used the term "evidence only" for this; it means that we deleted all the era's units and won't accept any new units, because we just keep the era around so we can verify evidence.

Not sure if "frozen" or "evidence only" is better?

Reply:
Definitely prefer "evidence-only".

Reply:
Done in https://github.com/casper-network/casper-node/pull/2127/commits/429a2b1af6acf011c0a141bc198e9bb52ecb75b8.